src/components/card/card.stories.mdx

```diff
@@ -528,6 +532,113 @@ Click on the card to see the effect
   </Story>
 </Preview>
 
+## With draggable
```

Comment:
This example does not work, nothing happens.

Please can you add some more detailed notes, how is this expected to work?